modules/tinymce/src/core/main/ts/init/Render.ts

```diff
@@ -17,7 +17,6 @@ import IconManager from '../api/IconManager';
 import NotificationManager from '../api/NotificationManager';
```

Comment:
This file has 2 errors according to the build:
```
08:50:11  modules/tinymce/src/core/main/ts/init/Render.ts(172,76): error TS2304: Cannot find name 'EditorSettings'.
08:50:11  modules/tinymce/src/core/main/ts/init/Render.ts(205,90): error TS2304: Cannot find name 'settings'
```
So I suspect you didn't update the changes made during the merge?

Reply:
I merged and fixed all merge errors, but forgot to check if it would actually build. My bad, that one.